src/Explorer.js

```diff
@@ -879,19 +903,63 @@ function defaultArgs(
 
 class FieldView extends React.PureComponent<FieldViewProps, {}> {
   _previousSelection: ?SelectionNode;
-  _addFieldToSelections = () =>
-    this.props.modifySelections([
-      ...this.props.selections,
-      this._previousSelection || {
-        kind: 'Field',
-        name: {kind: 'Name', value: this.props.field.name},
-        arguments: defaultArgs(
-          this.props.getDefaultScalarArgValue,
-          this.props.makeDefaultArg,
-          this.props.field,
-        ),
-      },
-    ]);
+  _addFieldToSelections = event => {
```

Comment:
The logic here is hard to follow because at a lot of subsequent steps, you have to check if you're trying to get all subselections or not.

Can you split the code into two cases, one where you press the alt key and one where you don't?

Reply:
Done